<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OFCleanCaseForm</class>
 <widget class="QWidget" name="OFCleanCaseForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Candidates in</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="dirname">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>dir</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="list_cand"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Please select actions:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_pack">
       <property name="text">
        <string>Pack before cleaning</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_clean_times">
       <property name="text">
        <string>Clean time steps</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_keep_first">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Keep first time</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_clean_post">
       <property name="text">
        <string>Clean postprocessing data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_clean_proc">
       <property name="text">
        <string>Clean processor directories</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_clean_all">
       <property name="text">
        <string>Clean everything</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_exec">
       <property name="text">
        <string>Execute</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
